docs/bricks_overview.rst

```diff
@@ -154,8 +154,8 @@ brick:
 
 When dealing with children, the life cycle actually becomes a bit more
 complicated. (The full life cycle is documented as part of the
-:class:`.Brick` class.) Before allocating or initializing parameters, the
-parent brick will call its :meth:`.Brick.push_allocation_config` and
+:class:`.Brick` class). Before allocating or initializing parameters, the
```

Comment:
Periods go inside the parentheses when it encloses a full sentence, as is the case here.